### DISCUSSION TITLE
Save Settings Button

### DISCUSSION THREAD

**Discussion by Witek-W** [AvaloniaUI/Avalonia#11206] [Q&A]
Hello, I have a SettingsWin.axaml in Views
`<ComboBox x:Name="BricksColorComboBox" Width="150" SelectedIndex="2">
                    <ComboBoxItem Content="Czerwony"/>
                    <ComboBoxItem Content="Niebieski"/>
                    <ComboBoxItem Content="Fioletowy"/>
                    <ComboBoxItem Content="Biały"/>
                </ComboBox>`
And Brick.cs in Models
` 
public Brick()
    {
        
        
        
        
        // Ustawiamy cegłe
        var random = new Random();
        var color = Color.FromRgb((byte)random.Next(256), (byte)random.Next(256), (byte)random.Next(256));
        var brush = new SolidColorBrush(color);

        this.Width = 79;
        this.Height = 30;
        
        //Ustawiamy kolor cegiełki
        
        this.Fill = Brushes.Purple;
        //Dodajemy obramowanie cegły
        
        this.StrokeThickness = 2;
        this.Stroke = new SolidColorBrush(Colors.Black);

        
    }`

I need help. I would like a button in settingswin.axaml that saves the selection from the combobox and redirects it to a variable to brick.cs. There I would like to do a switch() from this variable for example:

`switch(color){
case 0:
this.Fill = Brushes.Purple;
break;
case 1:
this.Fill = Brushes.White;
break;
etc.
}`

How to do it :((

**Comment by timunie** [comment]
@ErGrejt you should do some tutorials to get used to the MVVM approach. Doing this in code behind is a hard task imo. 

See: https://github.com/AvaloniaUI/Avalonia.Samples#mvvm-samples 
and for data presistence: https://docs.avaloniaui.net/guides/deep-dives/reactiveui/data-persistence
